custom_metrics/almanac.js

```diff
@@ -481,9 +481,9 @@ return JSON.stringify({
     // Gets metadata about the AMP plugin, if present.
     // Used by 14.2, 14.3, 14.4.
     try {
-      var metadata = document.querySelector('meta[name=generator][content^="AMP Plugin"]');
+      var metadata = document.querySelector("meta[name='generator'i][content^='AMP Plugin'i]");
       if (metadata) {
-        return metadata.getAttribute('content');
+        return metadata.getAttribute('content'); // this can now vary by case
```

Comment:
Do we really need this?
```suggestion
        return metadata.getAttribute('content');
```

Reply:
I'm altering someone else's data, so thought it would be good to include a clue if they had a problem?

Reply:
I think that’s what Git History is for so I’d remove it. It’s all of our code/data now since it’s a public repo 😀

Reply:
I'm still learning this world of cloud collaboration and version control. My poor memory and frequent need to reverse engineer probably means I comment a lot more than most. And that's when I'm working on my own code.